database/migrations/2021_08_18_134343_create_event_schedules_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateEventSchedulesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('event_schedules', function (Blueprint $table) {
+            $table->id();
+            $table->time('event_id');
+            $table->time('schedule_id');
```

Comment:
ここは外部キーが入るので型の変更と外部キー制約をいれてください〜！
```suggestion
            $table-> unsignedBigInteger('event_id');
            $table->foreign('event_id')->references('id')->on('events');
            $table-> unsignedBigInteger('schedule_id');
            $table->foreign('schedule_id')->references('id')->on('schedules');
```